### PR TITLE
Add RELEASE_VERSION build-arg to the image 📟

### PR DESCRIPTION
**Pull request by vdemeester**
# Changes

This will allow to build release version of the image, by default the
arg is empty, making it a `dev` version.

Signed-off-by: Vincent Demeester <vdemeest@redhat.com>

/cc @chmouel @danielhelfand 

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Run the code checkers with `make check`
- [ ] Regenerate the manpages, docs and go formatting with `make generated`
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/cli/blob/master/CONTRIBUTING.md)
for more details._

# Release Notes

### CODE REVIEW

File: contrib/tkn-image/Dockerfile [hunk 1]
```diff
@@ -2,9 +2,10 @@ ARG GOLANG_VERSION=1.13.8
 ARG DEBIAN_VERSION=10
 
 FROM golang:${GOLANG_VERSION} as builder
+ARG RELEASE_VERSION=
```
Comment (pradeepitm12) [inline contrib/tkn-image/Dockerfile]: Should we have latest as default?

Comment (vdemeester) [reply contrib/tkn-image/Dockerfile]: I'm letting the go code handling that. right now, it's setting it to `dev` if the given.

**Pull request merged**